### ISSUE TITLE
Building latest on mac

### ISSUE THREAD

**Issue by jpolton** [NOC-MSM/pyBDY#116]
```


[LIVMAZ:~/GitHub/pyBDY] jelt% conda env create -n pybdy -f environment.yml python=3.9
Collecting package metadata (repodata.json): done
Solving environment: done


==> WARNING: A newer version of conda exists. <==
  current version: 4.14.0
  latest version: 23.5.0

Please update conda by running

    $ conda update -n base -c conda-forge conda



Downloading and Extracting Packages
libwebp-1.3.0        | 77 KB     | ############################################################################################################################################### | 100% 
charset-normalizer-3 | 44 KB     | ############################################################################################################################################### | 100% 
lz4-c-1.9.4          | 153 KB    | ############################################################################################################################################### | 100% 
libxcb-1.15          | 306 KB    | ############################################################################################################################################### | 100% 
libaec-1.0.6         | 29 KB     | ############################################################################################################################################### | 100% 
numpy-1.24.3         | 6.0 MB    | ############################################################################################################################################### | 100% 
libglib-2.76.3       | 2.4 MB    | ############################################################################################################################################### | 100% 
libgfortran5-12.2.0  | 1.5 MB    | ############################################################################################################################################### | 100% 
fonttools-4.39.4     | 2.0 MB    | ############################################################################################################################################### | 100% 
tornado-6.3.2        | 615 KB    | ############################################################################################################################################### | 100% 
sip-6.7.9            | 466 KB    | ############################################################################################################################################### | 100% 
basemap-1.3.7        | 145 KB    | ############################################################################################################################################### | 100% 
curl-8.1.2           | 143 KB    | ############################################################################################################################################### | 100% 
libjpeg-turbo-2.1.5. | 448 KB    | ############################################################################################################################################### | 100% 
pip-23.1.2           | 1.3 MB    | ############################################################################################################################################### | 100% 
libcblas-3.9.0       | 14 KB     | ############################################################################################################################################### | 100% 
geos-3.11.2          | 1.2 MB    | ############################################################################################################################################### | 100% 
tzdata-2023c         | 115 KB    | ############################################################################################################################################### | 100% 
scipy-1.10.1         | 14.7 MB   | ############################################################################################################################################### | 100% 
netcdf4-1.6.4        | 425 KB    | ############################################################################################################################################### | 100% 
matplotlib-3.7.1     | 8 KB      | ############################################################################################################################################### | 100% 
packaging-23.1       | 45 KB     | ############################################################################################################################################### | 100% 
matplotlib-base-3.7. | 6.3 MB    | ############################################################################################################################################### | 100% 
basemap-data-1.3.2   | 24.1 MB   | ############################################################################################################################################### | 100% 
python-3.9.16        | 11.0 MB   | ############################################################################################################################################### | 100% 
importlib_resources- | 30 KB     | ############################################################################################################################################### | 100% 
readline-8.2         | 250 KB    | ############################################################################################################################################### | 100% 
lcms2-2.15           | 222 KB    | ############################################################################################################################################### | 100% 
importlib-resources- | 9 KB      | ############################################################################################################################################### | 100% 
gst-plugins-base-1.2 | 2.3 MB    | ############################################################################################################################################### | 100% 
pyqt-5.15.7          | 3.9 MB    | ############################################################################################################################################### | 100% 
wheel-0.40.0         | 54 KB     | ############################################################################################################################################### | 100% 
glib-2.76.3          | 470 KB    | ############################################################################################################################################### | 100% 
zstd-1.5.2           | 399 KB    | ############################################################################################################################################### | 100% 
libgfortran-5.0.0    | 156 KB    | ############################################################################################################################################### | 100% 
typing_extensions-4. | 34 KB     | ############################################################################################################################################### | 100% 
hdf4-4.2.15          | 708 KB    | ############################################################################################################################################### | 100% 
contourpy-1.0.7      | 204 KB    | ############################################################################################################################################### | 100% 
openssl-3.1.1        | 2.2 MB    | ############################################################################################################################################### | 100% 
requests-2.31.0      | 55 KB     | ############################################################################################################################################### | 100% 
libblas-3.9.0        | 14 KB     | ############################################################################################################################################### | 100% 
giflib-5.2.1         | 75 KB     | ############################################################################################################################################### | 100% 
libnetcdf-4.9.2      | 715 KB    | ############################################################################################################################################### | 100% 
snappy-1.1.10        | 34 KB     | ############################################################################################################################################### | 100% 
libllvm15-15.0.7     | 22.8 MB   | ############################################################################################################################################### | 100% 
blosc-1.21.4         | 49 KB     | ############################################################################################################################################### | 100% 
gstreamer-1.22.3     | 1.7 MB    | ############################################################################################################################################### | 100% 
setuptools-67.7.2    | 569 KB    | ############################################################################################################################################### | 100% 
mysql-libs-8.0.32    | 1.5 MB    | ############################################################################################################################################### | 100% 
llvm-openmp-16.0.5   | 289 KB    | ############################################################################################################################################### | 100% 
c-ares-1.19.1        | 101 KB    | ############################################################################################################################################### | 100% 
libclang13-15.0.7    | 6.7 MB    | ############################################################################################################################################### | 100% 
pyqt5-sip-12.11.0    | 73 KB     | ############################################################################################################################################### | 100% 
liblapack-3.9.0      | 14 KB     | ############################################################################################################################################### | 100% 
typing-extensions-4. | 10 KB     | ############################################################################################################################################### | 100% 
urllib3-2.0.2        | 95 KB     | ############################################################################################################################################### | 100% 
libopenblas-0.3.23   | 5.7 MB    | ############################################################################################################################################### | 100% 
pyproj-3.5.0         | 387 KB    | ############################################################################################################################################### | 100% 
zipp-3.15.0          | 17 KB     | ############################################################################################################################################### | 100% 
libpq-15.3           | 2.2 MB    | ############################################################################################################################################### | 100% 
libclang-15.0.7      | 130 KB    | ############################################################################################################################################### | 100% 
hdf5-1.14.0          | 3.3 MB    | ############################################################################################################################################### | 100% 
pillow-9.5.0         | 44.1 MB   | ############################################################################################################################################### | 100% 
pooch-1.7.0          | 50 KB     | ############################################################################################################################################### | 100% 
libssh2-1.11.0       | 253 KB    | ############################################################################################################################################### | 100% 
mysql-common-8.0.32  | 726 KB    | ############################################################################################################################################### | 100% 
python_abi-3.9       | 6 KB      | ############################################################################################################################################### | 100% 
nss-3.89             | 1.8 MB    | ############################################################################################################################################### | 100% 
qt-webengine-5.15.8  | 52.1 MB   | ############################################################################################################################################### | 100% 
qt-5.15.8            | 17 KB     | ############################################################################################################################################### | 100% 
sqlite-3.42.0        | 854 KB    | ############################################################################################################################################### | 100% 
libnghttp2-1.52.0    | 599 KB    | ############################################################################################################################################### | 100% 
qt-main-5.15.8       | 43.8 MB   | ############################################################################################################################################### | 100% 
glib-tools-2.76.3    | 95 KB     | ############################################################################################################################################### | 100% 
libdeflate-1.18      | 65 KB     | ############################################################################################################################################### | 100% 
libxml2-2.11.4       | 609 KB    | ############################################################################################################################################### | 100% 
libcurl-8.1.2        | 344 KB    | ############################################################################################################################################### | 100% 
libtiff-4.5.0        | 384 KB    | ############################################################################################################################################### | 100% 
proj-9.2.0           | 2.6 MB    | ############################################################################################################################################### | 100% 
libwebp-base-1.3.0   | 338 KB    | ############################################################################################################################################### | 100% 
platformdirs-3.5.1   | 18 KB     | ############################################################################################################################################### | 100% 
xorg-libxau-1.0.11   | 13 KB     | ############################################################################################################################################### | 100% 
libcxx-16.0.5        | 1.1 MB    | ############################################################################################################################################### | 100% 
icu-72.1             | 11.1 MB   | ############################################################################################################################################### | 100% 
libsqlite-3.42.0     | 858 KB    | ############################################################################################################################################### | 100% 
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
Installing pip dependencies: | Ran pip subprocess with arguments:
['/Users/jelt/opt/anaconda3/envs/pybdy/bin/python', '-m', 'pip', 'install', '-U', '-r', '/Users/jelt/GitHub/pyBDY/condaenv.ktoqnei9.requirements.txt']
Pip subprocess output:
Collecting pyjnius (from -r /Users/jelt/GitHub/pyBDY/condaenv.ktoqnei9.requirements.txt (line 1))
  Downloading pyjnius-1.5.0-cp39-cp39-macosx_10_9_x86_64.whl (271 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 271.7/271.7 kB 1.5 MB/s eta 0:00:00
Collecting seawater (from -r /Users/jelt/GitHub/pyBDY/condaenv.ktoqnei9.requirements.txt (line 2))
  Downloading seawater-3.3.4.tar.gz (22 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting thredds-crawler (from -r /Users/jelt/GitHub/pyBDY/condaenv.ktoqnei9.requirements.txt (line 3))
  Downloading thredds_crawler-1.5.4-py3-none-any.whl (10 kB)
Requirement already satisfied: numpy in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from seawater->-r /Users/jelt/GitHub/pyBDY/condaenv.ktoqnei9.requirements.txt (line 2)) (1.24.3)
Requirement already satisfied: requests in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from thredds-crawler->-r /Users/jelt/GitHub/pyBDY/condaenv.ktoqnei9.requirements.txt (line 3)) (2.31.0)
Collecting lxml (from thredds-crawler->-r /Users/jelt/GitHub/pyBDY/condaenv.ktoqnei9.requirements.txt (line 3))
  Downloading lxml-4.9.2-cp39-cp39-macosx_10_15_x86_64.whl (4.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.8/4.8 MB 3.9 MB/s eta 0:00:00
Collecting pytz (from thredds-crawler->-r /Users/jelt/GitHub/pyBDY/condaenv.ktoqnei9.requirements.txt (line 3))
  Downloading pytz-2023.3-py2.py3-none-any.whl (502 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 502.3/502.3 kB 2.2 MB/s eta 0:00:00
Requirement already satisfied: python-dateutil in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from thredds-crawler->-r /Users/jelt/GitHub/pyBDY/condaenv.ktoqnei9.requirements.txt (line 3)) (2.8.2)
Requirement already satisfied: six>=1.5 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from python-dateutil->thredds-crawler->-r /Users/jelt/GitHub/pyBDY/condaenv.ktoqnei9.requirements.txt (line 3)) (1.16.0)
Requirement already satisfied: charset-normalizer<4,>=2 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from requests->thredds-crawler->-r /Users/jelt/GitHub/pyBDY/condaenv.ktoqnei9.requirements.txt (line 3)) (3.1.0)
Requirement already satisfied: idna<4,>=2.5 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from requests->thredds-crawler->-r /Users/jelt/GitHub/pyBDY/condaenv.ktoqnei9.requirements.txt (line 3)) (3.4)
Requirement already satisfied: urllib3<3,>=1.21.1 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from requests->thredds-crawler->-r /Users/jelt/GitHub/pyBDY/condaenv.ktoqnei9.requirements.txt (line 3)) (2.0.2)
Requirement already satisfied: certifi>=2017.4.17 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from requests->thredds-crawler->-r /Users/jelt/GitHub/pyBDY/condaenv.ktoqnei9.requirements.txt (line 3)) (2023.5.7)
Building wheels for collected packages: seawater
  Building wheel for seawater (setup.py): started
  Building wheel for seawater (setup.py): finished with status 'done'
  Created wheel for seawater: filename=seawater-3.3.4-py3-none-any.whl size=26454 sha256=ca4e779cc05d47d38d8caba2946c26744ac8893f7aee27ce9919b4366f547583
  Stored in directory: /Users/jelt/Library/Caches/pip/wheels/ca/e5/4a/d04154df148b68a585471323bf5398366f93a994a384acbe9d
Successfully built seawater
Installing collected packages: pytz, pyjnius, seawater, lxml, thredds-crawler
Successfully installed lxml-4.9.2 pyjnius-1.5.0 pytz-2023.3 seawater-3.3.4 thredds-crawler-1.5.4

done
#
# To activate this environment, use
#
#     $ conda activate pybdy
#
# To deactivate an active environment, use
#
#     $ conda deactivate

Retrieving notices: ...working... done

```

**Comment by jpolton** [comment]
And 

```
[LIVMAZ:~/GitHub/pyBDY] jelt% source activate pybdy
activate: No such file or directory.
[LIVMAZ:~/GitHub/pyBDY] jelt% conda activate pybdy
(pybdy) [LIVMAZ:~/GitHub/pyBDY] jelt% 
(pybdy) [LIVMAZ:~/GitHub/pyBDY] jelt% echo $SHELL
/bin/bash
(pybdy) [LIVMAZ:~/GitHub/pyBDY] jelt% export JAVA_HOME=/Users/jelt/Downloads/jdk-20.0.1.jdk/Contents/Home 
export: Command not found.
(pybdy) [LIVMAZ:~/GitHub/pyBDY] jelt% setenv JAVA_HOME /Users/jelt/Downloads/jdk-20.0.1.jdk/Contents/Home
(pybdy) [LIVMAZ:~/GitHub/pyBDY] jelt% echo $JAVA_HOME
/Users/jelt/Downloads/jdk-20.0.1.jdk/Contents/Home
(pybdy) [LIVMAZ:~/GitHub/pyBDY] jelt% 
(pybdy) [LIVMAZ:~/GitHub/pyBDY] jelt% 
(pybdy) [LIVMAZ:~/GitHub/pyBDY] jelt% pip install -e .
Obtaining file:///Users/jelt/GitHub/pyBDY
  Installing build dependencies ... done
  Checking if build backend supports build_editable ... done
  Getting requirements to build editable ... done
  Installing backend dependencies ... done
  Preparing editable metadata (pyproject.toml) ... done
Requirement already satisfied: netCDF4 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from pyBDY==0.3.1.dev39+gfb49161) (1.6.4)
Requirement already satisfied: scipy in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from pyBDY==0.3.1.dev39+gfb49161) (1.10.1)
Requirement already satisfied: numpy in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from pyBDY==0.3.1.dev39+gfb49161) (1.24.3)
Requirement already satisfied: matplotlib in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from pyBDY==0.3.1.dev39+gfb49161) (3.7.1)
Requirement already satisfied: basemap in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from pyBDY==0.3.1.dev39+gfb49161) (1.3.7)
Requirement already satisfied: thredds-crawler in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from pyBDY==0.3.1.dev39+gfb49161) (1.5.4)
Requirement already satisfied: seawater in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from pyBDY==0.3.1.dev39+gfb49161) (3.3.4)
Requirement already satisfied: pyqt5 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from pyBDY==0.3.1.dev39+gfb49161) (5.15.7)
Requirement already satisfied: pyjnius in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from pyBDY==0.3.1.dev39+gfb49161) (1.5.0)
Requirement already satisfied: cftime in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from pyBDY==0.3.1.dev39+gfb49161) (1.4.1)
Requirement already satisfied: basemap-data<1.4,>=1.3.2 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from basemap->pyBDY==0.3.1.dev39+gfb49161) (1.3.2)
Requirement already satisfied: pyshp<2.4,>=1.2 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from basemap->pyBDY==0.3.1.dev39+gfb49161) (2.3.1)
Requirement already satisfied: pyproj<3.6.0,>=1.9.3 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from basemap->pyBDY==0.3.1.dev39+gfb49161) (3.5.0)
Requirement already satisfied: contourpy>=1.0.1 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from matplotlib->pyBDY==0.3.1.dev39+gfb49161) (1.0.7)
Requirement already satisfied: cycler>=0.10 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from matplotlib->pyBDY==0.3.1.dev39+gfb49161) (0.11.0)
Requirement already satisfied: fonttools>=4.22.0 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from matplotlib->pyBDY==0.3.1.dev39+gfb49161) (4.39.4)
Requirement already satisfied: kiwisolver>=1.0.1 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from matplotlib->pyBDY==0.3.1.dev39+gfb49161) (1.4.4)
Requirement already satisfied: packaging>=20.0 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from matplotlib->pyBDY==0.3.1.dev39+gfb49161) (23.1)
Requirement already satisfied: pillow>=6.2.0 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from matplotlib->pyBDY==0.3.1.dev39+gfb49161) (9.5.0)
Requirement already satisfied: pyparsing>=2.3.1 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from matplotlib->pyBDY==0.3.1.dev39+gfb49161) (3.0.9)
Requirement already satisfied: python-dateutil>=2.7 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from matplotlib->pyBDY==0.3.1.dev39+gfb49161) (2.8.2)
Requirement already satisfied: importlib-resources>=3.2.0 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from matplotlib->pyBDY==0.3.1.dev39+gfb49161) (5.12.0)
Requirement already satisfied: certifi in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from netCDF4->pyBDY==0.3.1.dev39+gfb49161) (2023.5.7)
Requirement already satisfied: PyQt5-sip<13,>=12.11 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from pyqt5->pyBDY==0.3.1.dev39+gfb49161) (12.11.0)
Requirement already satisfied: requests in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from thredds-crawler->pyBDY==0.3.1.dev39+gfb49161) (2.31.0)
Requirement already satisfied: lxml in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from thredds-crawler->pyBDY==0.3.1.dev39+gfb49161) (4.9.2)
Requirement already satisfied: pytz in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from thredds-crawler->pyBDY==0.3.1.dev39+gfb49161) (2023.3)
Requirement already satisfied: zipp>=3.1.0 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from importlib-resources>=3.2.0->matplotlib->pyBDY==0.3.1.dev39+gfb49161) (3.15.0)
Requirement already satisfied: six>=1.5 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from python-dateutil>=2.7->matplotlib->pyBDY==0.3.1.dev39+gfb49161) (1.16.0)
Requirement already satisfied: charset-normalizer<4,>=2 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from requests->thredds-crawler->pyBDY==0.3.1.dev39+gfb49161) (3.1.0)
Requirement already satisfied: idna<4,>=2.5 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from requests->thredds-crawler->pyBDY==0.3.1.dev39+gfb49161) (3.4)
Requirement already satisfied: urllib3<3,>=1.21.1 in /Users/jelt/opt/anaconda3/envs/pybdy/lib/python3.9/site-packages (from requests->thredds-crawler->pyBDY==0.3.1.dev39+gfb49161) (2.0.2)
Building wheels for collected packages: pyBDY
  Building editable for pyBDY (pyproject.toml) ... done
  Created wheel for pyBDY: filename=pyBDY-0.3.1.dev39+gfb49161-0.editable-py3-none-any.whl size=28781 sha256=e87fb57b6e26110cf06575511cf6e197a3eef41ee750bbb885794776a92cb3f7
  Stored in directory: /private/var/folders/11/r68qx1z14pn862rc3xk8xy_c470s2z/T/pip-ephem-wheel-cache-38c0nmk2/wheels/d5/54/63/4ed4f732587c035340ab64b84ff43ee57824d02a4cedc7676d
Successfully built pyBDY
Installing collected packages: pyBDY
Successfully installed pyBDY-0.3.1.dev39+gfb49161
(pybdy) [LIVMAZ:~/GitHub/pyBDY] jelt% 
(pybdy) [LIVMAZ:~/GitHub/pyBDY] jelt% 

```

**Comment by jdha** [comment]
Issues only arise on ARM64, issue being addressed by removing basemap #125
